packages/storebuilder/src/contexts/ShippingProvider.tsx

```diff
@@ -4,14 +4,13 @@ import ShippingScreenData, {
 } from '@shipping/data/shipping-screen-data';
 import { Confirmation, Error } from '@shipping/screens';
 import { handleActionRequest, removeNulls } from '@store/utils';
-import { SHIPPING_PROPS } from '@store/constants';
 
 export interface ShippingProviderContextInterface {
 	shippingState: ShippingScreenDataInterface;
 	setIsLoading: (isLoading: boolean) => void;
 	setShippingProviders: (providers: string[]) => void;
 	setProvidersActivated: (providersActivated: boolean) => void;
-	activatePlugins: () => void;
+	submitProvidersAndActivate: () => void;
```

Comment:
What do you mean by submit providers?

Reply:
Thanks for the note! I updated the function name to `activateShippingProviderPlugins` to make it better describe the intent of the function.